src/snowflake/connector/auth_keypair.py

```diff
@@ -47,7 +54,16 @@ def __init__(self, private_key, lifetime_in_seconds: int = LIFETIME):
         self._private_key = private_key
         self._jwt_token = ""
         self._jwt_token_exp = 0
-        self._lifetime = timedelta(seconds=lifetime_in_seconds)
+        self._lifetime = timedelta(
+            seconds=os.getenv("JWT_LIFETIME_IN_SECONDS", lifetime_in_seconds)
+        )
+        self._jwt_retry_attempts = os.getenv(
+            "JWT_CNXN_RETRY_ATTEMPTS", self.DEFAULT_JWT_RETRY_ATTEMPTS
```

Comment:
I think we should signify that this is a static variable by not using `self`
```suggestion
            "JWT_CNXN_RETRY_ATTEMPTS", AuthByKeyPair.DEFAULT_JWT_RETRY_ATTEMPTS
```